src/pages/resultsView/survival/SurvivalChart.tsx

```diff
@@ -410,10 +414,50 @@ export default class SurvivalChart extends React.Component<ISurvivalChartProps,
         return this.victoryLegendData.length > 0;
     }
 
+    @observable.shallow zoomDomain: any = {x: [0, this.initialRangeMaxValue]};
```

Comment:
you probably want @observable.ref if all you ever do is reset the reference

Reply:
@dippindots also I think it should be typed?